### DISCUSSION TITLE
Login via cli on wsl1

### DISCUSSION THREAD

**Discussion by ayitinya** [firecmsco/firecms#604] [Q&A] [answered]
When I try to login after `yarn create firecms-app` , I get the error in the browser `{"errno":-2,"code":"ENOENT","syscall":"open","path":"/tmp/xfs-f3e12df6/dlx-1968/node_modules/@firecms/cli/html/done.html"}`

it's a yarn workspaces set

```json
{
  "name": "project name",
  "workspaces": [
    "./apps/ui",
    "./apps/cms"
  ],
  "private": true,
  "packageManager": "yarn@3.6.3"
}
```

I am using WSL 1
Node 20

**Comment by fgatti675** [comment, marked as answer]
This should be fixed :)
